app/component/itinerary/TicketInformation.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+import ComponentUsageExample from '../documentation/ComponentUsageExample';
+
+export default function TicketInformation() {
+  return (
+    <div className="itinerary-ticket-information">
+      <div>
+        <FormattedMessage id="required-ticket" defaultMessage="Ticket required for the journey" />:
+        <div className="itinerary-ticket-information-class">Helsinki</div>
```

Comment:
Here we could prepare to show the ticket price, which will later on be dynamic, by making the text as in the design.